packages/@azure/eventhubs/client/package.json

```diff
@@ -19,13 +19,14 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/amqp-common": "^1.0.0-preview.2",
+    "@azure/ms-rest-nodeauth": "^0.9.1",
```

Comment:
Why do we need a direct dependency on `"@azure/ms-rest-nodeauth"`?
It only seems to be used in samples, in which case it should be a devDependency at most.
But since `amqp-common` already depends on this, we should be able to remove it